bitcoin/BRSyncManager.c

```diff
@@ -857,7 +857,7 @@ BRClientSyncManagerSubmit(BRClientSyncManager manager,
                                 BRClientSyncManagerAsSyncManager (manager),
                                 (BRSyncManagerEvent) {
                                     SYNC_MANAGER_TXN_SUBMITTED,
-                                    { .submitted = {transaction->txHash, -1} },
+                                    { .submitted = {transaction, -1} },
```

Comment:
The function declaration specifies `transaction` as `OwnershipKept` - does this event callback take the transaction?

This:
```
typedef void
(*BRSyncManagerEventCallback) (void * context,
                               BRSyncManager manager,
                               OwnershipKept BRSyncManagerEvent event);
```
implies that the contents of `event` are themselves kept?